rpc/backend/tracing.go

```diff
@@ -246,7 +246,7 @@ func (b *Backend) TraceCall(
 	}
 
 	// minus one to get the context of block beginning
-	contextHeight := blk.Block.Height - 1
+	contextHeight := blk.Block.Height
```

Comment:
the minus one is to use the state of the beginning of the block, do you just want to change the block number in evm context?

Reply:
this is to avoid [w3_wait_for_new_blocks(w3, 1, sleep=0.1)](https://github.com/crypto-org-chain/ethermint/pull/391/files#diff-3e148d436d65c5cd83194e9ed01b8f4c97931ce2410fe2bdde6e840b6f14965cL437) before and won't use [req.BlockNumber](https://github.com/crypto-org-chain/ethermint/blob/develop/x/evm/keeper/grpc_query.go#L551) at last?